src/matchmaker/match.js

```diff
@@ -1,3 +1,6 @@
+// TODO: use hash?
```

Comment:
Is there any advantage using hashvalue?
Increasing the matchid after every match created should create an unique id for every match as well right?!

Reply:
For now there is no issue, the only issue would be that you don't want to have duplicates for instance when restarting the app and a tournament is already running..